app/src/main/res/drawable/all_infocenter_dot.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:fillColor="?android:attr/textColorPrimary"
+      android:pathData="M17.912,1.002A3,3 0,1 0,21 4a3,3 0,0 0,-3.088 -2.998zM6,2c-1.1,0 -2,0.9 -2,2v16c0,1.1 0.9,2 2,2h12c1.1,0 2,-0.9 2,-2L20,8.578a5,5 0,0 1,-2 0.418L18,20L6,20L6,4h1v9l3,-2.25L13,13L13,4a5,5 0,0 1,0.424 -2L6,2zM9,4h2v5l-1,-0.75L9,9L9,4z"/>
```

Comment:
What would you think about coloring the dot red?

```suggestion
    <path
        android:fillColor="?android:attr/textColorPrimary"
        android:pathData="M6,2c-1.1,0 -2,0.9 -2,2v16c0,1.1 0.9,2 2,2h12c1.1,0 2,-0.9 2,-2L20,8.578a5,5 0,0 1,-2 0.418L18,20L6,20L6,4h1v9l3,-2.25L13,13L13,4a5,5 0,0 1,0.424 -2L6,2zM9,4h2v5l-1,-0.75L9,9L9,4z" />
    <path
        android:fillColor="#FFF44336"
        android:pathData="M17.912,1.002A3,3 0,1 0,21 4a3,3 0,0 0,-3.088 -2.998z" />
```

Reply:
That wouldn't work because the icons get tinted.

Reply:
I see. That's unfortunate.